scripts/seed_data.py

```
@@ -0,0 +1,109 @@
+            data = json.load(file)
+    except FileNotFoundError:
+        print(f"❌ Error: {file_path} not found")
+        return
+    except json.JSONDecodeError:
+        print(f"❌ Error: Invalid JSON format in {file_path}")
+        return
+
+    try:
+        create_entities(data)
+        print("🎉 Data loaded successfully")
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)